packages/grib/grib.0.11.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer: "hez@0ok.org"
+authors: [ "Hezekiah M. Carty" ]
+license: "MIT"
+homepage: "https://github.com/hcarty/ocaml-grib"
+bug-reports: "https://github.com/hcarty/ocaml-grib/issues"
+dev-repo: "https://github.com/hcarty/ocaml-grib.git"
+tags: [ "clib:png" "clib:openjpeg" "clib:m" "clib:jpeg" "clib:jasper"
+        "clib:grib" "weather" "grib" ]
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install:["ocaml" "setup.ml" "-install"]
+remove: ["ocamlfind" "remove" "grib"]
+depends: [
+  "bear"
+  "ocamlfind" {build}
+]
+available: [ ocaml-version >= "4.02.1" ]
+depexts: [
+  [
+    ["debian"]
+    ["libjpeg-dev" "libjasper-dev" "libopenjpeg-dev" "libgrib-api-dev" "libpng-dev"]
+  ]
+  [
+    ["ubuntu"]
+    ["libjpeg-dev" "libjasper-dev" "libopenjpeg-dev" "libgrib-api-dev" "libpng-dev"]
+  ]
```

Comment:
Thank you for the suggestion.  I should have access to an osx machine soon. I'll try to test and add the appropriate package names at that point.